_includes/live-demos/url-conversion-absolute-1/index.js

```diff
@@ -1,7 +1,7 @@
 tinymce.init({
   selector: '#url-conversion-absolute-1',
-  height: 230,
+  height: 300,
```

Comment:
How come we changed the height here?

Reply:
because 230 leads to a scrollbar, 300 on these url demos eliminate the scrollbar so you can see the demo content "at a glance"